app/components/team/Row.js

```diff
@@ -41,7 +41,7 @@ class TeamRow extends React.PureComponent {
       <Panel.RowLink key={this.props.team.id} to={`/organizations/${this.props.team.organization.slug}/teams/${this.props.team.slug}`}>
         <div className="flex flex-stretch items-center line-height-1" style={{ minHeight: '3em' }}>
           <div className="flex-auto">
-            <div className="m0 flex items-center">
+            <div className="m0 flex items-center mb1">
```

Comment:
But line-height effects text wrapping… I think I'll just add a conditional instead, to apply the `mb1` if there's a description.